src/Microsoft.AspNetCore.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -15,7 +15,6 @@ namespace Microsoft.AspNetCore.Razor.TagHelpers
     public class TagHelperOutput : IHtmlContentContainer
     {
         private readonly Func<bool, HtmlEncoder, Task<TagHelperContent>> _getChildContentAsync;
-        private TagHelperAttributeList _attributes;
         private TagHelperContent _preElement;
         private TagHelperContent _preContent;
         private TagHelperContent _content;
```

Comment:
Wondering if we still need lazy-init for any of the properties in this class? Probably fine not to churn the code but, if lazy isn't needed, should we bother -- especially for `_content`? @rynowak said he was fine either way and I'm feeling ambivalent as well as I type this. But let's confirm the choice.